vulnerabilities/importer_yielder.py

```diff
@@ -25,7 +25,7 @@
 IMPORTER_REGISTRY = [
     {
         'name': 'rust',
-        'license': 'https://creativecommons.org/publicdomain/zero/1.0/',
+        'license': 'CC0-1.0',
         'last_run': None,
         'data_source': 'RustDataSource',
```

Comment:
I think we discussed it before ... IMHO I would prefer to have a real class rather than a class name.